src/main/java/com/github/manolo8/darkbot/utils/DiscordWebhook.java

```diff
@@ -0,0 +1,238 @@
+package com.github.manolo8.darkbot.utils;
+
+import com.github.manolo8.darkbot.utils.http.Http;
+import com.github.manolo8.darkbot.utils.http.Method;
+import com.google.gson.Gson;
+import com.google.gson.annotations.SerializedName;
+
+import java.awt.*;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Adaptation of original source: https://gist.github.com/k3kdude/fba6f6b37594eae3d6f9475330733bdb
+ */
+public class DiscordWebhook {
+    private static final Gson GSON = new Gson();
+    private final List<EmbedObject> embeds = new ArrayList<>();
+    private String content;
+    private String username;
+    @SerializedName("avatar_url") private String avatarUrl;
+    private boolean tts;
```

Comment:
```suggestion
    private static final Gson GSON = new Gson();
    
    private String content;
    private String username;
    @SerializedName("avatar_url") private String avatarUrl;
    private boolean tts;
    private final List<EmbedObject> embeds = new ArrayList<>();
```

It seems like the order of the fields & getters/setters is a bit over the place, it'd be nice of both the fields and the getters/setters followed the order in https://discord.com/developers/docs/resources/webhook#execute-webhook